internal/database/repository/node.go

```diff
@@ -7,8 +7,27 @@ import (
 	"github.com/apache/yunikorn-core/pkg/webservice/dao"
 	"github.com/jackc/pgx/v5"
 	"github.com/oklog/ulid/v2"
+
+	"github.com/G-Research/yunikorn-history-server/internal/database/sql"
 )
 
+type NodeUtilizationFilters struct {
```

Comment:
What would you think about changing all occurrences of `NodeUtilizationXXX` to just `NodeUtilXXX`? I notice that the YK core uses the shorter style, like in `dao.PartitionNodesUtilDAOInfo`?   Besides being closer to the YK core style, it would be much less typing!

Reply:
Updated. I left some cz for those utilization sounded better. Let me know what do you think